darts/utils/statistics.py

```diff
@@ -305,9 +305,53 @@ def plot_acf(ts: TimeSeries,
     axis.plot((0, max_lag + 1), (0, 0), color='black')
 
 
+def plot_hist(data: Union[TimeSeries, List[float], np.ndarray],
+              bins: Optional[Union[int, np.ndarray, List[float]]] = None,
+              density: bool = False,
+              title: Optional[str] = None,
+              fig_size: Optional[Tuple[int, int]] = None,
+              ax: Optional[plt.axis] = None) -> None:
+    """ This function plots the histogram of `data`.
+
+    Parameters
+    ----------
+    data
+        Univariate TimeSeries instance from which to plot the histogram.
+    bins
+        Optionally, either an integer value for the number of bins to be displayed
+        or an array-like of floats determining the position of bins.
+    density
+        bool, if `density` is set to True, the bin counts will be converted to probability density
+    title
+        The title of the figure to be displayed
+    fig_size
+        The size of the figure to be displayed.
+    ax
+        Optionally, an axis object to plot the histogram on.
+    """
+
+    raise_if_not(any([isinstance(data, x) for x in [TimeSeries, list, np.ndarray]]),
+                 'input data must be a TimeSeries instance or an array-like')
+
+    if isinstance(data, TimeSeries):
+        data._assert_univariate()
+        values = data.univariate_values() if data.is_deterministic else data.all_values().flatten()
```

Comment:
I think simply having `values = data.all_values().flatten()` should work in all cases.

Actually, I'm thinking that you could remove the requirement for the series to be univariate, and plot several histograms (maybe on the same figure) with different colors/labels for multivariate series. One histogram per component - and maybe only plot the top-5 or something like this if the series contains too many components (you can raise a warning if this happens).

Reply:
I updated it now with https://github.com/unit8co/darts/pull/477/commits/51da4dee1f1fbb0d4042de36776f21583b772c93 that if the TimeSeries contains more than one component, it plots up to four histograms of the first four components.

If `data` is a list or np.ndarray I will simply flatten the values as I think that is the major use case.